tests/test_scenario.py

```diff
@@ -548,6 +548,136 @@ def test_clone(self, session, network_with_data):
 
         return updated_network
 
+
+    def test_scenario_inheritance(self, session, network_with_data):
```

Comment:
Maybe split this into multiple tests for clarity?

Reply:
Done